LuisPalominoTrevilla/TaskManagerSystem
# Accounts Microservice

## Changes

- Created structure for accounts microservice
- Made module for handling AWS S3 file downloads and uploads
- Made module to edit local files
- Created servlet for accounts
- Changed accounts documentation
- Added additional endpoints

## Prerequisites

- Refer to the README for software requirements information. 
- You will ned an env file, send me a private message

## Test

- Run the documentation **Refer to the instructions inside the README**
- Run the accounts microservice **Refer to the README instructions inside the accounts folder**
- Most of the endpoints (except for the last two) should be fully functional
